README.md

```diff
@@ -99,4 +99,49 @@ $ docker-compose run bintest
 * [mruby-eso-research](https://github.com/hone/mruby-eso-research) - an app for managing crafting research in Elder Scrolls Online. It uses YAML as the data store.
 
 ## mruby-cli Development
-This app is built as a mruby-cli app, so you just need to run: `docker-compose run compile` and find the binaries in the appropriate directories.
+
+### Compile the mruby-cli binaries
+
+This app is built as a `mruby-cli` app, so you just need to run:
+
+```
+docker-compose run compile
+```
+
+and find the binaries in the appropriate directories.
+
+#### Common error
+
+If rather than typing `docker-compose run compile`, you type `rake compile`, you could get on a nix OS the following error:
+
+```
+/usr/include/features.h:364:25: fatal error: sys/cdefs.h: No such file or directory
+ #  include <sys/cdefs.h>
+
+ compilation terminated.
+ In file included from /usr/include/string.h:25:0,
+                  from /tmp/mruby-cli/mruby/src/dump.c:7:
+ /usr/include/features.h:364:25: fatal error: sys/cdefs.h: No such file or directory
+  #  include <sys/cdefs.h>
+                          ^
+ compilation terminated.
+ rake aborted!
+ Command failed with status (1): [gcc -g -std=gnu99 -O3 -Wall -Werror-implic...]
+
+ ...
+```
+
+### Create the releases
+
+Just type: `rake release`
```

Comment:
The release task is meant to be run from the container, https://github.com/hone/mruby-cli/blob/50ce2dc11036f0733d1aa9e27c6b1a1e64edb7dd/docker-compose.yml#L21-L23

> Did you try it on other kind of host?
No, but it assumes gnu tar and other things. They exist on the container but would potentially break in OS X? The container will create the files anyways, so that's good enough. There is the root problem in linux, but not sure how to fix that with Docker.

> That discussion makes me wondering the following: "why not calling docker-compose from ad hoc rake task?"
Not sure what you mean by this.